### PR TITLE
Improved time to first paint 

### PR DESCRIPTION
**Pull request by naveed-ahmad**
intl pollyfil should included before our main js, opening pr for testing the impact of moving js into bottom.

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:33176

File: src/helpers/Html.js [hunk 1]
```diff
@@ -79,6 +79,9 @@ const Html = ({ store, component, assets }) => {
         {Object.keys(assets.javascript).map((script, i) =>
           <script src={assets.javascript[script]} key={i} />
         )}
+        {
+          <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css" />
```
Comment (mmahalwy) [inline src/helpers/Html.js]: hmm i thought `link`s should always be in head?

Comment (naveed-ahmad) [inline src/helpers/Html.js]: No it can be anywhere in page. We can move all no critical css and js to bottom of page to improve rendering

Comment (mmahalwy) [inline src/helpers/Html.js]: I think we should remove font-awesome since I think it's only used in one place

**Pull request merged**